content/clause-interleaving.md

```diff
@@ -1,8 +1,9 @@
 ---
 layout: sip
-title: SIP-NN Clause Interleaving
-stage: design
-permalink: /sips/:title.html
+title: SIP-47 Clause Interleaving
+stage: implementation
+status: waiting-for-implementation
```

Comment:
Shouldn't we change that AFTER the vote? 😅

Reply:
/cc @julienrf, WDYT?

Reply:
It makes sense to change before the vote: the Committee votes for moving the proposal to the “implementation” stage.